h/services/__init__.py

```diff
@@ -20,6 +20,7 @@ def includeme(config):
     config.register_service_factory('.flag.flag_service_factory', name='flag')
     config.register_service_factory('.flag_count.flag_count_service_factory', name='flag_count')
     config.register_service_factory('.group.groups_factory', name='group')
+    config.register_service_factory('.group_create.group_create_factory', name='group_create')
```

Comment:
We've got some existing services named `{verb}_{object}` eg. `list_groups`, `delete_group` as well as some services named `{object}_{related_object}` eg. `group_link`, `group_member`. `group_create` looks inconsistent on this basis. It will help future developers if we can try to settle on what convention(s) we are using for service naming.

Reply:
I wanted to have a conversation about that. My personal two recommendations would be:

* Subdirectories/submodules: e.g. we'd not have a flat `services` directory. This is my preference because I like clear groupings of things and short modules with small areas of responsibilities.
* `<resource>_<verb>`. When I'm scanning modules, I think resource first, then verb. Having a bunch of things grouped by verb is not as useful to me. I typically want to find all of the services pertaining to a particular resource, not have to find them scattered about. The naming here matches this convention.

Reply:
I'd like @seanh input on this ^^.

Reply:
:+1: I think lots of small modules and lots of small directories, and `<resource>_<verb>` all sounds great. Within reason I think we want to proactively try to make our directories and modules (and classes and methods and functions) as small and simple as possible, and with good organisation and naming, not just let them grow and only split them up when they're really definitely _too_ big.